spec/api/people/show_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+require "spec_helper"
+
+RSpec.describe "people#show", type: :request do
+  let(:token) { service_tokens(:permitted_root_layer_token).token }
+  let(:person) { people(:admin) }
+
+  subject(:make_request) { jsonapi_get "/api/people/#{person.id}" }
+
+  before do
+    person.update!(sac_remark_national_office: "Remark", sac_remark_section_1: "Remark")
+    sign_in(person)
```

Comment:
Shouldn't you use the token instead of signing in? I think you should be using `it_behaves_like "jsonapi authorized requests" do` like in the other  api specs.

Reply:
i don't know how the token is supposed to work, but that'd be better